### PR TITLE
sensor url param throw a warning

### PR DESCRIPTION
**Pull request by adelamarre**
Don not add sensor url param when SENSOR is false or null.

### CODE REVIEW

**Comment by adelamarre** [comment]
Hello ? Anybody here ? Up ;)

File: lib/Google.js [hunk 1]
```diff
@@ -107,9 +107,9 @@
 		var url = GoogleMapsLoader.URL;
 
 		url += '?callback=' + GoogleMapsLoader.WINDOW_CALLBACK_NAME;
-
-		url += '&sensor=' + ((GoogleMapsLoader.SENSOR === true || GoogleMapsLoader.SENSOR === 'true') ? 'true' : 'false');
-
+		if(GoogleMapsLoader.SENSOR) {
```
Comment (davidkudera) [inline lib/Google.js]: Can you please add this variable into other properties? Just like in this [pull request](https://github.com/Carrooi/Js-GoogleMapsLoader/pull/28/files#diff-faf7c0808229ee587378c07a22d69f84R53).

Also missing white space after `if`

File: lib/Google.js [hunk 2]
```diff
@@ -107,9 +107,9 @@
...
+			url += '&sensor=' + ((GoogleMapsLoader.SENSOR === true || GoogleMapsLoader.SENSOR === 'true') ? 'true' : 'false');
+		}
```
Comment (davidkudera) [inline lib/Google.js]: Missing new line after `}`

**Comment by davidkudera** [comment]
Hi and sorry for such human-timeout :confounded:....

There is already some other pull request (#28) but it seems to be quite dead, so if you'll just please fix those typos, I'll definitelly merge it :wink:. Oh, and after squashing those 3 commits together.

PS: happy new year

**Comment by scriptist** [comment]
The sensor should be removed entirely, not just if false.

https://developers.google.com/maps/documentation/javascript/error-messages

> The sensor parameter is no longer required for the Google Maps JavaScript API. It won't prevent the Google Maps JavaScript API from working correctly, but we recommend that you remove the sensor parameter from the script element.

**Comment by davidkudera** [comment]
@scriptist :+1:

**Comment by danbovey** [comment]
+1 for this. It's the only thing in my console across my whole site. So merge asap please! :smiley:

**Comment by davidkudera** [comment]
@danbovey I will gladly do that after this merge request is ready :wink:

**Comment by scriptist** [comment]
I've created a new PR with the correct changes: #34

**Comment by davidkudera** [comment]
Closing in favor of #34 
Thank you anyway :+1:

**Pull request closed without merging**